schemas/session-replay/_common-shape-wireframe-schema.json

```diff
@@ -16,6 +16,7 @@
           "properties": {
             "backgroundColor": {
               "type": "string",
+              "pattern": "^[#][a-f0-9]{2}[a-f0-9]{2}[a-f0-9]{2}([a-f0-9]{2})?",
               "description": "The background color for this wireframe as a String hexadecimal. Follows the #RRGGBBAA color format, default value should be #FFFFFF00.",
```

Comment:
Description says `#RRGGBBAA`, but the pattern allows for `#RRGGBB` too - it's inconsistent. Also, the pattern is missing end-of-string `$`. I think we can simplify it to:
```
"pattern": "^#[a-f0-9]{8}$"
```
no?

Reply:
actually we decided that we will have the alpha optional.I will modify the description to state that.